encoder/src/main/java/openlr/encoder/routesearch/SecondShortestRouteChecker.java

```diff
@@ -0,0 +1,192 @@
+package openlr.encoder.routesearch;
+
+import openlr.encoder.OpenLREncoderProcessingException;
+import openlr.map.GeoCoordinates;
+import openlr.map.Line;
+import openlr.map.utils.GeometryUtils;
+import openlr.map.utils.PQElem;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * <h1>Verifying the existence of Fully or partially joined second shortest route with length less than relative threshold</h1>
+ */
+public class SecondShortestRouteChecker {
+    private GeoCoordinates destination;
+    private List<? extends Line> location;
+    private int maxLengthAllowed;
+    private double relativeThreshold;
+
+
```

Comment:
please remove overall unnecessary empty line. Let only one and not two

Reply:
00c70cf1bd4f3e32f47bccb0523ebe37cbbce6e8